lib/routes/watasuke/blog.js

```diff
@@ -0,0 +1,44 @@
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import cheerio from 'cheerio';
+
+const baseUrl = 'https://watasuke.net';
+
+export default async (ctx) => {
+    const blogPageHtml = await got(`${baseUrl}/blog/`).text();
+    const $ = cheerio.load(blogPageHtml);
+    const ogTitle = $('meta[property="og:title"]').attr('content');
+
+    const apiUrl = `${baseUrl}/page-data/blog/page-data.json`;
+    const response = await got(apiUrl);
+    const articles = response.data.result.data.allArticles.nodes;
+
+    const itemsPromises = articles.map(async (article) => {
+        const articleUrl = `${baseUrl}/blog/article/${article.slug}`;
```

Comment:
Please use the redirected url instead.
logs: https://github.com/DIYgod/RSSHub/actions/runs/8128222519/job/22213891885#step:12:56